src/richie/apps/persons/factories.py

```diff
@@ -58,46 +59,35 @@ def title(self):
 
     @factory.post_generation
     # pylint: disable=unused-argument
-    def with_content(self, create, extracted, **kwargs):
+    def fill_portrait(self, create, extracted, **kwargs):
         """
-        Add content plugins displayed in the "portrait" and "resume" placeholder
-        of the person page:
-
-        - Picture plugin featuring a random portrait image,
-        - Text plugin featuring the person resume with a random long text.
+        Add a portrait with a random image
         """
         if create and extracted:
-            language = settings.LANGUAGE_CODE
-            portrait_placeholder = self.extended_object.placeholders.get(
-                slot="portrait"
-            )
-
-            resume_placeholder = self.extended_object.placeholders.get(slot="resume")
+            portrait_placeholder = self.extended_object.placeholders.get(slot="portrait")
 
-            # Add a portrait with a random image
             portrait_file = file_getter(os.path.dirname(__file__), "portrait")()
             wrapped_portrait = File(portrait_file, portrait_file.name)
             portrait = Image.objects.create(file=wrapped_portrait)
-            add_plugin(
-                language=language,
-                placeholder=portrait_placeholder,
-                plugin_type="PicturePlugin",
-                picture=portrait,
-                attributes={"alt": "portrait image"},
-            )
-
-            # Add a text plugin for resume with a long random text
-            nb_paragraphs = random.randint(2, 4)
-            paragraphs = [
-                factory.Faker("text", max_nb_chars=random.randint(200, 1000)).generate(
-                    {}
+            for language in self.extended_object.get_languages():
+                add_plugin(
+                    language=language,
+                    placeholder=portrait_placeholder,
+                    plugin_type="PicturePlugin",
+                    picture=portrait,
+                    attributes={"alt": "portrait image"},
                 )
-                for i in range(nb_paragraphs)
-            ]
-            body = ["<p>{:s}</p>".format(p) for p in paragraphs]
-            add_plugin(
-                language=language,
-                placeholder=resume_placeholder,
-                plugin_type="CKEditorPlugin",
-                body="".join(body),
+
+    @factory.post_generation
+    # pylint: disable=unused-argument
+    def fill_resume(self, create, extracted, **kwargs):
```

Comment:
Isn't a good opportunity to put fill_texts() method from https://github.com/openfun/richie/blob/master/src/richie/apps/courses/factories.py#L210 in common Factory mixin so we can have the same logic to create text plugin share everywhere ?

Reply:
@sveetch that's enticing but maybe not in this PR? For the person bio, I have set "nb_paragraphs=1" so we need to rethink how to allow that if we factorize the code.